app/components/map-utility-box.js

```diff
@@ -61,24 +62,24 @@ export default Ember.Component.extend({
 
     return [
       {
-        label: `${format(stops[7][0])} or more`,
-        color: stops[7][1],
+        label: `${format(stops[3])} or more`,
+        color: colors[4],
```

Comment:
I'm confused by this - how can we always expect the 5 index of the colors array to be what we want here?

Reply:
Colors array is fixed-length, and all choropleths have a fixed number of breaks.  If we ever deviate from this, the code above won't work.

Reply:
Okay. You can also destructure those too:
`const [first, second, third, fourth, fifth, sixth, seventh] = stops;`